### DISCUSSION TITLE
New 9fans.net/go/draw/frame package

### DISCUSSION THREAD

**Discussion by fhs** [rjkroege/edwood#323] [General]
A new frame package and sam port got pushed out today by Russ:
https://pkg.go.dev/9fans.net/go@v0.0.4/draw/frame
https://pkg.go.dev/9fans.net/go@v0.0.4/cmd/sam

Haven't looked deep into the frame package yet, but it would be nice to converge on one frame package.

**Comment by rjkroege** [comment]
I'll look into it. Excellent suggestion.
